pebblo/app/service/prompt/prompt_service.py

```diff
@@ -72,7 +72,7 @@ def _add_retrieval_data(self, retrieval_data):
             logger.error(message)
             return return_response(message=message, status_code=500)
 
-        retrieval_data["ai_app"] = ai_app_obj.id
+        retrieval_data["ai_app"] = ai_app_obj[0].id
```

Comment:
Lets have defensive code to avoid any error due to hard coded index.

Reply:
Updated this code block